### ISSUE TITLE
one last export date is saved for all books

### ISSUE THREAD

**Issue by nikopp** [codinguser/gnucash-android#580]
#### Expected behaviour
Last export date is saved per book.

#### Actual behaviour
One last export date is saved for all books.

#### Steps to reproduce the behaviour
1. Have more than one book.
2. Export Book 1.
3. Book 1 and Book 2 have the same (new) export date.

#### Software specifications
* GnuCash Android version: 2.1.1 beta
* System Android version: 6.0.1
* Device type: Sony Xperia Z3 compact

**Comment by rivaldi8** [comment]
I can't seem to reproduce the issue, I see each book with its export date. Also the code looks correct.

You refer to the "since" date that appears in the export form, don't you? Which format are you exporting to? These books were created in a previous version of GnuCash or did you create them in the one you report?

**Comment by nikopp** [comment]
> You refer to the "since" date that appears in the export form, don't you?

No. I just checked and that is ok. Sorry to be unclear about that.

I refer to the "manage books" section that can be selected via the menu and the drop down for the books (last entry). All books are listed there together with the number of accounts and the number of transactions. The last line is the date of the last export and it is the same for all books.

> Which format are you exporting to?

QIF

> These books were created in a previous version of GnuCash or did you create them in the one you report?

They were created with the one I reported.
